schema_forms/mammo_form.py

```diff
@@ -9,24 +9,25 @@ class MammoArchDistortionsForm(FlaskForm):
     class Meta:
         csrf = False
 
-
-    mammo_arch_location_right_breast = SelectField("Location of Architectural Distortion on Right Breast", choices =
-    MammographyDict.mammo_arch_location_right_breast_choice)
-    mammo_arch_location_left_breast = SelectField("Location of Architectural Distortion on Left Breast", choices =
-                                                  MammographyDict.mammo_arch_location_left_breast_choice)
-    mammo_arch_depth =  SelectField("Depth of Architectural Distortion", choices =
+    architectural_distortions_location_right_breast = SelectField("Location of Architectural Distortion on Right Breast",
```

Comment:
arch_dist_loc_right not good enough?

Reply:
loc_right may be ok, but I think I just wanted the prefix to be the same because I thought that's how it would club them together..